be/src/vec/exec/scan/new_olap_scanner.cpp

```diff
@@ -385,6 +385,23 @@ Status NewOlapScanner::_init_return_columns() {
     return Status::OK();
 }
 
+doris::TabletStorageType NewOlapScanner::get_storage_type() {
+    int local_reader = 0;
+    for (const auto& reader : _tablet_reader_params.rs_readers) {
+        if (reader->rowset()->rowset_meta()->resource_id().empty()) {
```

Comment:
// 3. remote scan thread pool
    _remote_scan_thread_pool.reset(
            new PriorityThreadPool(config::doris_scanner_thread_pool_thread_num,
                                   config::doris_scanner_thread_pool_queue_size, "remote_scan"));

maybe we should add another config for remote scanner thread pool. because we may increase the remote scanner thread pool to 10000 and it is not the same as local scanner thread pool.

Reply:
done